presto-main/src/main/java/io/prestosql/execution/scheduler/NodeScheduler.java

```diff
@@ -156,6 +159,7 @@ public NodeSelector createNodeSelector(ConnectorId connectorId)
                 }
                 catch (UnknownHostException e) {
                     // ignore
+                    LOG.debug(e, "Unable to resolve host name for: " + node);
```

Comment:
`.error`? `.warn`?

"for:" -> "for node:"

Reply:
Use strong formatting. This constructs the string even if the log level is disabled.